### ISSUE TITLE
[NPCs] Daily (! and ?) issue list 

### ISSUE THREAD

**Issue by Echinaceax** [WoWManiaUK/Blackwing-Lair#5214]
**Whats happening**

Daily quest giver should be blue quest pickups/deliveries  (not yellow)
letting players know its a repeatable daily quest and to show up on their minimap

**Links:**

- [x] old-man-barlo https://www.wowhead.com/npc=25580/old-man-barlo#starts
- [x] Lord Torvos http://cata.cavernoftime.com/npc=25140 for https://cata-twinhead.twinstar.cz/?quest=11877
- [x] Commander Kunz http://cata.cavernoftime.com/npc=28039 / https://github.com/WoWManiaUK/Blackwing-Lair/issues/4952 (closed and added here)
- [x] trapper-maui https://www.wowhead.com/npc=26228/trapper-maui / https://github.com/WoWManiaUK/Blackwing-Lair/issues/5180 (closed and added here) for daily https://www.wowhead.com/quest=11960/planning-for-the-future
- [x] utaik  https://www.wowhead.com/npc=26213/utaik / https://github.com/WoWManiaUK/Blackwing-Lair/issues/5179 (closed and added here) for quest https://www.wowhead.com/quest=11945/preparing-for-the-worst
- [x] Wind Trader Zhareem https://cata-twinhead.twinstar.cz/?npc=24369 for https://cata-twinhead.twinstar.cz/?quest=11369

 **Isle of Quel'Danas**
- [x] Nether-Stalker Mah'duun https://cata-twinhead.twinstar.cz/?npc=24370 for https://cata-twinhead.twinstar.cz/?quest=11385
- [x] Harbinger Haronem https://cata-twinhead.twinstar.cz/?npc=19475 for https://cata-twinhead.twinstar.cz/?quest=11880
- [x] Exarch Nasuun https://cata-twinhead.twinstar.cz/?npc=24932 for https://cata-twinhead.twinstar.cz/?quest=11514
- [x] Captain Theris Dawnhearth https://cata-twinhead.twinstar.cz/?npc=24967 for https://cata-twinhead.twinstar.cz/?quest=11523
- [x] Magister Ilastar https://cata-twinhead.twinstar.cz/?npc=25069 for https://cata-twinhead.twinstar.cz/?quest=11540
- [x] Captain Valindria https://cata-twinhead.twinstar.cz/?npc=25088 for https://cata-twinhead.twinstar.cz/?quest=11541
- [x] Smith Hauthaa https://cata-twinhead.twinstar.cz/?npc=25046 for https://cata-twinhead.twinstar.cz/?quest=11536
- [x] Vindicator Xayann https://cata-twinhead.twinstar.cz/?npc=24965 for https://cata-twinhead.twinstar.cz/?quest=11525
- [x] Vindicator Kaalan https://cata-twinhead.twinstar.cz/?npc=25108 for https://cata-twinhead.twinstar.cz/?quest=11543
- [x] Astromancer Darnarian https://cata-twinhead.twinstar.cz/?npc=25133 for https://cata-twinhead.twinstar.cz/?quest=11547
- [x] Mar'nah https://cata-twinhead.twinstar.cz/?npc=24975 for https://cata-twinhead.twinstar.cz/?quest=11546
- [x] Battlemage Arynna https://cata-twinhead.twinstar.cz/?npc=25057 for https://cata-twinhead.twinstar.cz/?quest=11533
- [x] Harbinger Inuuro https://cata-twinhead.twinstar.cz/?npc=25061 for https://cata-twinhead.twinstar.cz/?quest=11537
- [x] Anchorite Ayuri https://cata-twinhead.twinstar.cz/?npc=25112 for https://cata-twinhead.twinstar.cz/?quest=11548

**Blade edge**
- [x] Magistrix Seyla https://cata-twinhead.twinstar.cz/?npc=24937 for https://cata-twinhead.twinstar.cz/?quest=11516 and https://cata-twinhead.twinstar.cz/?quest=11515

**other issues** 
this daily isnt given https://cata-twinhead.twinstar.cz/?quest=11875 after given back the required quest https://cata-twinhead.twinstar.cz/?quest=11550

**Comment by mfranca0009** [comment]
This whole issue is fixed 👍 

 https://cata-twinhead.twinstar.cz/?quest=11550 was completely broken, neither the NPC that handed it out or the NPC that takes the turn-in was setup, therefore the quest was never available. This has been fixed ✔️ 
 
 https://cata-twinhead.twinstar.cz/?quest=11875 never had a prerequisite assigned to 11550, but I have added this now ✔️ Also, the reason why this daily is not available to everyone is because you're required to have master mining, master herbalism, or master skinning to take this quest from Emissary Mordin. The details of these conditions are stated within its quest details text from WoWhead or the link above at twinstar.
